CONTRIBUTING.md

```diff
@@ -2,39 +2,47 @@
 
 ## Coordination, communication and community
 
-Many ember-paper contributor hang out on the [ember-paper channel on slack](https://embercommunity.slack.com/messages/ember-paper/). Not a slack user? [Get your invitation.](https://ember-community-slackin.herokuapp.com/)
+Many ember-paper contributors hang out on the [ember-paper channel on slack](https://embercommunity.slack.com/messages/ember-paper/). Not a slack user? [Get your invitation.](https://ember-community-slackin.herokuapp.com/)
 
 ## Coding style
 
-* ember-paper uses the [ember-suave](https://github.com/DockYard/ember-suave) coding style.
+* **Suave.** ember-paper uses the [ember-suave](https://github.com/DockYard/ember-suave) coding style.
 
-* Before submitting a pull request,
+* **jscs.** Before submitting a pull request,
 check for coding style issues with  `jscs -c .jscsrc app addon`.
 
-* Require action closures rather than strings representing action names.
+* **Actions.** Require action closures rather than strings representing action names.
 `{{some-component someAction=(action "myAction")}}`, not `{{some-component someAction="myAction"}}`.
 
-## Capitalization and naming
+* **Encapsulation.** When communicating with a private ember-paper component, bind as many properties as are needed.
+When communicating between two public ember components, use `nearestOfType` to find the outer component,
```

Comment:
> use `nearestOfType` to find the outer component
might be a bit too strong because of contextual components?

A better description could be:
> When communicating between two public ember components, use `nearestOfType` in a computed property to find the outer component (for pre 2.3 compatibility), and override it when yielding a contextual component.

When we have an example of that, we can link it here as well.